unittest/python_pybind11/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.4)
+project(test_boost_pybind11 LANGUAGES CXX)
+
+find_package(pybind11 REQUIRED)
+find_package(pinocchio REQUIRED)
```

Comment:
I do think this test should be inserted within the main Pinocchio tests, by loading the Pybind11 module from github